plugins/@grouparoo/spec-helper/src/lib/specHelper.ts

```diff
@@ -10,6 +10,7 @@ const {
 const corePath: string = getCoreRootPath();
 process.chdir(corePath);
 process.env.ACTIONHERO_CONFIG = `${corePath}/${loadPath}/config`;
+process.env.GROUPAROO_SPEC_HELPER = "true";
```

Comment:
While not functional, this will serve as a reminder that the specHelper has modified the ENV and CWD